assets/messages.yml

```diff
@@ -856,6 +856,21 @@ messages:
         {quote}Problems related to modifying internal NBT tags during the lifetime of an entity are not considered bugs.
         -- [~searge]'s comment [here|https://bugs.mojang.com/browse/MC-66943?focusedCommentId=188032&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-188032] in MC-66943.{quote}
 
+        %quick_links%
+      fillname: []
+  modified-world-crash:
+    - project:
+        - mc
+        - mcl
+        - mctest
+      name: Modified world crash
+      shortcut: worldmod
```

Comment:
This will collide with the existing `shortcut: world` (for `attach-world-file`). Does anyone have a better suggestion?

Reply:
`mod-world` maybe?

Reply:
I think that in turn would collide with `shortcut: mod` from `modified-game`, see also #152 for why this is a problem.

Maybe either the shortcut for either `attach-world-file` or `modified-game` could be changed though.

Reply:
I think `mworld` would be best